ui/components/connections/index.js

```diff
@@ -560,13 +611,14 @@ function Connections({
    * fetch connections when the page loads
    */
   useEffect(() => {
-    getConnections(page, pageSize, search);
+    getConnections(page, pageSize, search, sortOrder);
```

Comment:
should'nt sortOrder also be in dependencies

Reply:
if i am right changes are handled in onTableChangeAlso  also .

Reply:
also is the sort sustained when move to next pages across al changed columns?